File: Segundo Semestre/LAB121/Proyecto/Arte.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Proyecto
{
    public class Arte
    {
        private int AñodePublicacion;
        public int añodePublicacion
        {
            get { return AñodePublicacion; }
            set { AñodePublicacion = value; }
        }
        private string NombredelaObra;
        public string nombredelaObra
        {
            get { return NombredelaObra; }
            set { NombredelaObra = value; }
        }
        public void Leer()
        {
            System.Console.WriteLine("Leer el año de publicacion de la obra: ");
            añodePublicacion = int.Parse(Console.ReadLine());
            System.Console.WriteLine("Leer el nombre de la obra: ");
            NombredelaObra = Console.ReadLine();
        }
        public void Mostrar()
        {
            System.Console.WriteLine(AñodePublicacion + " " + NombredelaObra);
        }
    }
}

File: Segundo Semestre/LAB121/JavaInterfaz/src/Arte.java

    public class Arte
    {
        private int AñodePublicacion;

    /*public Arte(int AñodePublicacion, String NombredelaObra) {
        this.AñodePublicacion = AñodePublicacion;
        this.NombredelaObra = NombredelaObra;
    }*/

    public String getNombredelaObra() {
        return NombredelaObra;
    }

    public int getAñodePublicacion() {
        return AñodePublicacion;
    }
        /*public int añodePublicacion
        {
            get { return AñodePublicacion; }
            set { AñodePublicacion = value; }
        }*/
        private String NombredelaObra;
        private String Popularidad;
        private String Aceptacion;
        /*public String nombredelaObra
        {
            get { return NombredelaObra; }
            set { NombredelaObra = value; }
        }*/
        /*public void Leer()
        {
            System.out.println("Leer el año de publicacion de la obra: ");
            añodePublicacion = int.Parse(Console.ReadLine());
            System.out.println("Leer el nombre de la obra: ");
            NombredelaObra = Console.ReadLine();
        }
        public void Mostrar()
        {
            System.out.println(AñodePublicacion + " " + NombredelaObra);
        }*/

    public String getPopularidad() {
        return Popularidad;
    }

    public String getAceptacion() {
        return Aceptacion;
    }

    public Arte(int AñodePublicacion, String NombredelaObra, String Popularidad, String Aceptacion) {
        this.AñodePublicacion = AñodePublicacion;
        this.NombredelaObra = NombredelaObra;
        this.Popularidad = Popularidad;
        this.Aceptacion = Aceptacion;
    }
    }
